crates/spirv-std/src/textures.rs

```diff
@@ -172,6 +172,48 @@ impl Image2dArray {
             result
         }
     }
+    #[spirv_std_macros::gpu_only]
+    pub fn sample_by_lod(&self, sampler: Sampler, coordinate: Vec3A, lod: f32) -> Vec4 {
```

Comment:
This needs to be rebased onto the latest main and use the traits instead of glam types.